book_maker/obok.py

```diff
@@ -0,0 +1,865 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Version 4.1.1 March 2023
+# Make obok.py works as file selector
+
+# Version 4.1.0 February 2021
+# Add detection for Kobo directory location on Linux
```

Comment:
Comments in the first its a vender file.

Reply:
Sorry, I don't understand. You want me to remove shebang?

Reply:
sorry what I mean since its a vender file
We can add something like

```suggestion
# the original code from …
```

Reply:
Good idea!

Reply:
Resolved in e5b5687